packages/util-logger/src/util-logger.ts

```diff
@@ -62,7 +69,7 @@ const buildLoggerMessageArgs = ({title, message}) => {
  * 
  */
 export const log = async ({title, message, level, always = false}) => {
-  const configLoggerLevel = await config.get("logger.level", LEVELS.warn)
+  const configLoggerLevel = getLoggerLevel()
```

Comment:
Using `process.env` for the logger level is a breaking change and additionally `process.env` doesn't exist in browser environments.  Is there a reason we need to change this?